setup.cfg

```diff
@@ -4,3 +4,19 @@ style = pep440
 versionfile_source = pygeos/_version.py
 versionfile_build = pygeos/_version.py
 tag_prefix =
+
+[flake8]
+ignore = E203, E266, E501, W503, "pygeos/_version.py"
+
+[isort]
+atomic = true
+force_alphabetical_sort = true
```

Comment:
I think you should add: `sections=FUTURE,STDLIB,FIRSTPARTY,THIRDPARTY,LOCALFOLDER`
([source](https://pycqa.github.io/isort/docs/configuration/custom_sections_and_ordering/))

alphebetical sort within those categories would be good though.

Reply:
Thanks, I just copied these settings from another project. In the meantime, isort added `profile=black` which fixes most of the things. Also I changed the alphabetic sorting to alphabetic sorting *per section* (as you suggested)